src/NCPopupMenu.cc

```diff
@@ -58,7 +72,8 @@ NCPopupMenu::NCPopupMenu( const wpos & at, YItemIterator begin, YItemIterator en
 
 NCPopupMenu::~NCPopupMenu()
 {
-    itemsMap.clear();
+    for (auto item : _items)
```

Comment:
Please do **not** use `auto` for variable types: This defeats the purpose of having strict type checking in C++.